plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveS3AndGlueMetastoreTest.java

```diff
@@ -115,6 +113,16 @@ protected void validateFilesAfterOptimize(String location, Set<String> initialFi
         assertThat(getAllDataFilesFromTableDirectory(location)).isEqualTo(updatedFiles);
     }
 
+    private String adaptTableLocation(String location)
+    {
+        if (location.endsWith("/")) {
+            //Hive removes trailing slash from location
+            location = location.substring(0, location.length() - 1);
+        }
+        //Hive normalizes double slash
```

Comment:
nit: space after //

Reply:
indeed. (that was copied, thanks for catching)